diff --git a/dist/index.mjs b/dist/index.mjs
index 5af2e62651583fdf41c3453cb77d2d236c8be9f1..3d702c383f97576996290ac8cb698d5f83f282fa 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -1,6 +1,6 @@
 import { onBackPress, onHide, onInit, onLoad, onReady, onShow, onUnload, onPageScroll } from '@dcloudio/uni-app';
-import { ref, computed, getCurrentInstance, getCurrentScope, onScopeDispose, reactive, shallowRef, isRef, readonly, unref, watch, onMounted } from 'vue';
-import { resolveUnref, watchWithFilter, until, tryOnScopeDispose as tryOnScopeDispose$1, resolveRef, useIntervalFn, tryOnMounted, pausableWatch } from '@vueuse/core';
+import { ref, computed, getCurrentInstance, getCurrentScope, onScopeDispose, reactive, shallowRef, isRef, readonly, unref, watch, onMounted, toValue, toRef } from 'vue';
+import { watchWithFilter, until, tryOnScopeDispose as tryOnScopeDispose$1, useIntervalFn, tryOnMounted, pausableWatch } from '@vueuse/core';
 
 const name = "@uni-helper/uni-use";
 
@@ -456,7 +456,7 @@ function useActionSheet(options) {
     return uni.showActionSheet(
       reactive(__spreadValues$b(__spreadValues$b({
         itemList: []
-      }, resolveUnref(options)), resolveUnref(newOptions)))
+      }, toValue(options)), toValue(newOptions)))
     );
   };
 }
@@ -791,7 +791,7 @@ function useGlobalData(initialValue, options = {}) {
     eventFilter
   } = options;
   const app = ref(getApp());
-  const rawInit = resolveUnref(initialValue);
+  const rawInit = toValue(initialValue);
   const data = (shallow ? shallowRef : ref)(initialValue);
   watchWithFilter(data, () => {
     var _a;
@@ -841,7 +841,7 @@ function hideLoading() {
 function useLoading(options) {
   function showLoading(newOptions) {
     uni.showLoading(
-      reactive(__spreadValues$8(__spreadValues$8({}, resolveUnref(options)), resolveUnref(newOptions)))
+      reactive(__spreadValues$8(__spreadValues$8({}, toValue(options)), toValue(newOptions)))
     );
     return hideLoading;
   }
@@ -880,7 +880,7 @@ var __spreadValues$7 = (a, b) => {
 function useModal(options) {
   return function showModal(newOptions) {
     return uni.showModal(
-      reactive(__spreadValues$7(__spreadValues$7({}, resolveUnref(options)), resolveUnref(newOptions)))
+      reactive(__spreadValues$7(__spreadValues$7({}, toValue(options)), toValue(newOptions)))
     );
   };
 }
@@ -1030,7 +1030,7 @@ function useProvider(options) {
     return uni.getProvider(
       reactive(__spreadValues$6(__spreadValues$6({
         service: "oauth"
-      }, resolveUnref(options)), resolveUnref(newOptions)))
+      }, toValue(options)), toValue(newOptions)))
     );
   };
 }
@@ -1270,7 +1270,7 @@ var __spreadValues$4 = (a, b) => {
 function useScanCode(options) {
   return function scanCode(newOptions) {
     return uni.scanCode(
-      reactive(__spreadValues$4(__spreadValues$4({}, resolveUnref(options)), resolveUnref(newOptions)))
+      reactive(__spreadValues$4(__spreadValues$4({}, toValue(options)), toValue(newOptions)))
     );
   };
 }
@@ -1410,7 +1410,7 @@ function useSocket(url, options = {}) {
   const isConnecting = computed(() => status.value === "CONNECTING");
   const isClosed = computed(() => status.value === "CLOSED");
   const taskRef = ref();
-  const urlRef = resolveRef(url);
+  const urlRef = toRef(url);
   let heartbeatPause;
   let heartbeatResume;
   let explicitlyClosed = false;
@@ -1615,7 +1615,7 @@ function useStorage(key, initialValue, options = {}) {
     initOnMounted,
     storage = uni
   } = options;
-  const rawInit = resolveUnref(initialValue);
+  const rawInit = toValue(initialValue);
   const type = guessSerializerType(rawInit);
   const hasStore = !!store[key];
   const data = hasStore ? store[key] : (shallow ? shallowRef : ref)(rawInit);
@@ -1889,7 +1889,7 @@ var __spreadValues$1 = (a, b) => {
 function useToast(options) {
   return function showToast(newOptions) {
     uni.showToast(
-      __spreadValues$1(__spreadValues$1({}, resolveUnref(options)), resolveUnref(newOptions))
+      __spreadValues$1(__spreadValues$1({}, toValue(options)), toValue(newOptions))
     );
     return function hideToast() {
       return uni.hideToast();
